### ISSUE TITLE
[Mate] setup the accessibility environment variables automatically after install

### ISSUE THREAD

**Issue by king-dahmanus** [linuxmint/mint21-beta#150]
**Describe the bug**
it's not a bug, but rather a feature request. Please have the environment variables for accessibility preconfigured so that blind people don't have to look for them, I am not sure if I'll find them again. This is not necessarily only for mate, but the mate edition is the most accessible for blind people so I put that at the top
**To Reproduce**
as I said before, this isn't a bug, so no steps, only the variables
**Expected behavior**
this will make qt apps and electron apps more accessible when used with orca. Here are the variables that you should enable by default :
export ACCESSIBILITY_ENABLED=1
export GTK_MODULES=gail:atk-bridge
export GNOME_ACCESSIBILITY=1
export QT_ACCESSIBILITY=1
export QT_LINUX_ACCESSIBILITY_ALWAYS_ON=1
export DBUS_SESSION_BUS_PID
export DBUS_SESSION_BUS_ADDRESS
**Screenshots**

**Hardware and mintUpdate**
* If relevant, you could provide `inxi -Fxxxrz` output to describe your hardware and drivers
* Is your system all up-to-date ? Please check your Update Manager (mintUpdate).

**Additional context**
thanks for helping and I hope you consider my suggestion

**Comment by clefebvre** [comment]
Aren't these set by MATE when you enable A11Y? If not this should be reported to MATE directly. We can't set these for everybody out of the box as this would enable a11y if I'm not mistaken.

I'm closing the issue here in any case since it's not a regression and we're trying to get STABLE out.

**Comment by king-dahmanus** [comment]
no it won't enable the accessibility for everyone, come on please don't act on haste. I mentioned before that it's not only in mate. It only improves app accessibility when orca is enabled

**Comment by king-dahmanus** [comment]
or I have a suggestion. Instead of auto enabling it by default, you could have a script that activates after orca is enabled, thereby enabling it only when necessary
